renderer/reducers/lnd.js

```diff
@@ -352,10 +385,16 @@ export const fetchSeedError = error => dispatch => {
 }
 
 /**
- * Create a new wallet.
+ * Create a new wallet using settings from the onboarding state.
+ * @param  {Object} options Options
+ * @param  {Object} options.recover Boolean indicating weather this is a recovery
+ * @return {Promise}
  */
-export const createNewWallet = () => async (dispatch, getState) => {
-  dispatch({ type: CREATE_NEW_WALLET })
+export const createWallet = ({ recover } = {}) => async (dispatch, getState) => {
+  dispatch({ type: CREATE_WALLET })
+
+  const grpcHelpers = await getGrpcHelpers()
```

Comment:
can we move this inside try?

Reply:
No, because it's used in the catch block